### DISCUSSION TITLE
I have use dpdata generate box.raw  coord.raw, how can I generate energy.npy and force.npy ?

### DISCUSSION THREAD

**Discussion by tonystarkiss** [deepmodeling/dpdata#183] [Q&A]
(base) root@d7f92b70f22c:/data/examples/dpmd_raw# ls
box.raw  coord.raw  type.raw  type_map.raw
(base) root@d7f92b70f22c:/data/examples/dpmd_raw# /data/data/raw/raw_to_set.sh 
nframe is 1
nline per set is 2000
will make 1 sets
making set 0 ...
(base) root@d7f92b70f22c:/data/examples/dpmd_raw# ls
box.raw  coord.raw  set.000  type.raw  type_map.raw
(base) root@d7f92b70f22c:/data/examples/dpmd_raw# ll set.000/
total 16
drwxr-xr-x 2 root root 4096 Jul 30 09:32 ./
drwxr-xr-x 3 root root 4096 Jul 30 09:32 ../
-rw-r--r-- 1 root root  164 Jul 30 09:32 box.npy
-rw-r--r-- 1 root root  224 Jul 30 09:32 coord.npy

**Comment by njzjz** [comment]
You need to provide `energy.raw` and `force.raw`.
